src/lib/types.ts

```diff
@@ -1,17 +1,38 @@
-enum State {
-  Received,
-  Production,
-  Complete
-}
+type State = "received" | "production" | "completed" | "dispatched";
 
 class Order {
-  id: number;
+  id: string;
   state: State;
+  collectionId: string;
+  collectionName: string;
+  created: Date;
+  customer: string;
+  drinks: Array<any>;
+  paymentFulfilled: boolean;
+  updated: Date;
 
-  constructor(id: number, state = State.Received) {
-    this.id = id;
-    this.state = state;
+  constructor(data: {
+    id: string;
+    state: string;
+    collectionId: string;
+    collectionName: string;
+    created: string;
+    customer: string;
+    drinks: Array<any>;
```

Comment:
Hva mangler for at vi skal kunne bruke pocketbase types?

Reply:
...en ny PR? Jeg valgte å ikke inkludere det i denne PR gitt at kravet ikke er funksjonelt.